manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ossia.score
+PackageVersion: 3.1.4
+PackageLocale: en-US
+Publisher: ossia
+PackageName: score

```

Comment:
```suggestion
PackageURL: https://ossia.io/
PackageName: score
```

Reply:
Thank you for your guidance on this and my other submissions!